tools/bin/hawq_ctl

```diff
@@ -387,6 +387,10 @@ class HawqInit:
                  (self.GPHOME, self.standby_host_name, self.GPHOME)
         check_return_code(remote_ssh(scpcmd, self.master_host_name, self.user), \
                           logger, "Sync slaves file failed")
+        scpcmd = "scp %s/ranger/etc/* %s:%s/ranger/etc/ > /dev/null" % \
+                 (self.GPHOME, self.standby_host_name, self.GPHOME)
+        check_return_code(remote_ssh(scpcmd, self.master_host_name, self.user), \
+                          logger, "Sync rps configuration files failed")
```

Comment:
This code snippet is used for copy ranger configuration files to standby when do "hawq init standby" when ranger was configured.  But there maybe no ranger.  Standalone case will fail.
e.g. When hawq init cluster(it will init standby), hawq is absolutely not configured with ranger. If there is a cluster with hawq master and segment configured with ranger, then we should scp rps configuration files to standby when "hawq init standby".